external_libraries/amgcl/mpi/distributed_matrix.hpp

```diff
@@ -4,7 +4,7 @@
 /*
 The MIT License
 
-Copyright (c) 2012-2019 Denis Demidov <dennis.demidov@gmail.com>
+Copyright (c) 2012-2020 Denis Demidov <dennis.demidov@gmail.com>
```

Comment:
2021?

Reply:
next time we update the amgcl lib this will appear, but we should not change an header we don't author